admin/class-yoast-form.php

```diff
@@ -512,10 +513,7 @@ public function select( $var, $label, array $select_options, $styled = 'unstyled
 		$select = new Yoast_Input_Select( $var, $select_name, $select_options, $active_option );
 		$select->add_attribute( 'class', 'select' );
 
-		if (
-			$this->is_control_disabled( $var )
-			|| ( isset( $attr['disabled'] ) && $attr['disabled'] )
-		) {
+		if ( $this->is_control_disabled( $var ) || ( isset( $attr['disabled'] ) && $attr['disabled'] ) ) {
```

Comment:
Not sure about this change: splitting the conditions in two lines was more readable and it conforms to our coding standards.

Reply:
Changed it back!

Reply:
Changed back!